tool/site/client/src/services/test-packs/index.ts

```diff
@@ -44,31 +52,33 @@ export const createPrescriptionsFromExcelFile = (
 }
 
 function createPrescriptions(
+  prescriptionType: PrescriptionType,
   patients: Array<fhir.BundleEntry>,
   prescribers: Array<fhir.BundleEntry>,
-  nominatedPharmacies: Array<string>,
+  places: Array<fhir.BundleEntry>,
   rows: Array<PrescriptionRow>,
   setLoadPageErrors: Dispatch<SetStateAction<any>>
 ): any[] {
   const prescriptions = []
-  const groupedPrescriptionRows = groupBy(rows, (row: PrescriptionRow) => row.testId)
-  groupedPrescriptionRows.forEach(prescriptionRows => {
+  const prescriptionRows = groupBy(rows, (row: PrescriptionRow) => row.testId)
+
+  prescriptionRows.forEach(prescriptionRows => {
```

Comment:
could we rename either the first or second `prescriptionRows` in `prescriptionRows.forEach(prescriptionRows => {`? It's a bit confusing which is being referred to